### PR TITLE
[Call transfer] transfer storybook updates

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

update Transfer page in storybook
# Why

provide documentation to contoso about the new transfer feature

# How Tested

inspected locally (checkout the storybook link)

### CODE REVIEW

File: packages/storybook/stories/AdhocCalling.stories.mdx [hunk 1]
```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
...
 
-A call can be started with a Teams user or a call queue by using their id. The id of the Teams user or call queue is
-added to the `CallParticipantsLocator` which is used as the locator in the
-[AzureCommunicationCallAdapterArgs](./?path=/docs/composite-adapters--page#azurecommunicationcalladapterargs)
-when creating your [CallAdapter](path=/docs/composite-adapters--page#call-adapter).
-In the current beta version, only one Teams user can be called. A Teams call queue can also be called. For more
-information on call queues, refer to this
-[Learn documentation](https://learn.microsoft.com/en-us/microsoftteams/create-a-phone-system-call-queue)
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or, multiple teams users. You cannot call multiple
```
Comment (edwardlee-msft) [inline packages/storybook/stories/AdhocCalling.stories.mdx:19]:
```suggestion
like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application or, multiple teams users. You cannot call multiple
```

**Review by edwardlee-msft — APPROVED** [review]

File: packages/storybook/stories/AdhocCalling.stories.mdx (hunk 1 above)
Comment (mgamis-msft) [reply packages/storybook/stories/AdhocCalling.stories.mdx:19]: Grammar police time. I think it should be '...like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or multiple teams users. You cannot call multiple...'

File: packages/storybook/stories/AdhocCalling.stories.mdx [hunk 2]
```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
 
 # Adhoc calling
```
Comment (mgamis-msft) [inline packages/storybook/stories/AdhocCalling.stories.mdx:14]: I should have done it a long time ago but could you change all instances of 'adhoc' to 'ad-hoc' in this doc?

File: packages/storybook/stories/AdhocCalling.stories.mdx [hunk 3]
```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
...
+<Source code={AdhocCallingVoiceAppSnippetText} />
 
 The two callback functions `onFetchProfile` and `onFetchAvatarPersonaData` provided at the bottom of the snippet are
 for customizing display names and avatars of users, respectively.
 
+## Calling a Teams user
+
+<SingleLineBetaBanner />
+
+Like a Teams voice application you can call a Teams user by using their id. The id of the Teams user should be stored in the type of `MicrosoftTeamsUserIdentifier[]`.
```
Comment (mgamis-msft) [inline packages/storybook/stories/AdhocCalling.stories.mdx:43]: missing comma. 'Like a Teams voice application, you can call a Teams user by using their id'

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**